.github/workflows/e2e-wpuf.yml

```
@@ -0,0 +1,122 @@
+
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: "7.3"
+            
+      - uses: actions/checkout@v3
+        # with:
+        #   ref: 'automation/wpuf-e2e-pw'
+
+     
+      - uses: actions/setup-node@v3
```

actionlint (1.7.4)

the runner of "actions/setup-node@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)